src/PrestaShopBundle/Entity/Repository/AttributeRepository.php

```diff
@@ -84,11 +84,17 @@ public function findByLangAndShop($idLang, $idShop)
 
     private function getAttributeRow($attribute)
     {
-        return array(
+         $attributes = array(
             'id' => $attribute['id'],
             'color' => $attribute['color'],
             'position' => $attribute['attributePosition'],
             'name' => $attribute['attributeName'],
+            'texture' => '',
         );
+        if (@filemtime(_PS_COL_IMG_DIR_ . $attribute['id'] . '.jpg')) {
```

Comment:
In this case both of them return the needed response, but you are right about that file_exist is more suitable, I will change it
Thanks !